Makefile

```
@@ -1,7 +1,7 @@
 .PHONY: codegen-format update-version
 update-version:
 	@echo "$(VERSION)" > VERSION
-	@perl -pi -e 's|"version": "[.\d]+"|"version": "$(VERSION)"|' package.json
+	@perl -pi -e 's|"version": "[.\d\w-]+"|"version": "$(VERSION)"|' package.json
```

```suggestion
	@perl -pi -e 's|"version": "[.\-\d\w]+"|"version": "$(VERSION)"|' package.json
```
Very nitty, feel free to skip. I had to think for a sec what's \w- in regex ;)